internal/venice-client-common/src/main/java/com/linkedin/venice/pushmonitor/ExecutionStatus.java

```diff
@@ -88,26 +88,55 @@ public enum ExecutionStatus {
   /** Job/Task is created but ingestion haven't started yet */
   NOT_STARTED(false, true, false, false, 16),
 
-  DATA_RECOVERY_COMPLETED(false, true, false, false, 17);
+  DATA_RECOVERY_COMPLETED(false, true, false, false, 17),
+
+  /** DaVinci client specific ERRORS: rootStatus => ERROR */
+  /** DaVinci client fails ingestion due to disk reaching the threshold in the host */
+  DVC_INGESTION_ERROR_DISK_FULL(true, false, true, true, ERROR, 18),
+
+  /** DaVinci client fails ingestion due to reaching the configured memory limit in the host */
+  DVC_INGESTION_ERROR_MEMORY_LIMIT_REACHED(true, false, true, true, ERROR, 19),
+
+  /** There are too many dead DaVinci instances leading to failed push job */
+  DVC_INGESTION_ERROR_TOO_MANY_DEAD_INSTANCES(true, false, true, true, ERROR, 20),
+
+  /** Other uncategorized DaVinci Client errors */
+  DVC_INGESTION_ERROR_OTHER(true, false, true, true, ERROR, 21);
 
   final boolean isJobStatus;
   final boolean isTaskStatus;
-  final boolean isUsedByHybridStoresOnly;
+  final boolean isUsedByDaVinciClientOnly;
```

Comment:
Why this change?
Do we use this field: `isUsedByHybridStoresOnly` today?

Reply:
We do have a few `true` for the existing enums and they are not DaVinci only:
```
  /** Tasks belonging to a Hybrid Store emits this instead of {@link ExecutionStatus#COMPLETED} when it consumes a EOP message */
  END_OF_PUSH_RECEIVED(true, true, true, false, 4),
  /** Tasks belonging to a Hybrid Store emits this instead when it consumes a SOBR message */
  @Deprecated
  START_OF_BUFFER_REPLAY_RECEIVED(false, true, true, false, 5),
  /** Tasks belonging to a Hybrid Store emits this instead when it consumes a TS message */
  TOPIC_SWITCH_RECEIVED(false, true, true, false, 6),

CATCH_UP_BASE_TOPIC_OFFSET_LAG(false, true, true, false, 13),
```

With this renaming, I assume we should update the 3rd param to be `false`, right?

Reply:
Thanks for catching this. Fixed it and added small tests for these fields to reduce copy paste errors in the future.